include/circt/Conversion/Passes.td

```diff
@@ -0,0 +1,39 @@
+//===-- Passes.td - Conversion pass definitions ------------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file contains definitions for all dialect conversions.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_CONVERSION_PASSES_TD
+#define CIRCT_CONVERSION_PASSES_TD
+
+include "mlir/Pass/PassBase.td"
+
+//===----------------------------------------------------------------------===//
+// FIRRTLToRTL
+//===----------------------------------------------------------------------===//
+
+def LowerFIRRTLToRTLModule
+  : Pass<"lower-firrtl-to-rtl-module", "mlir::ModuleOp"> {
+  let summary = "Lower FIRRTL to RTL Modules";
+  let description = [{
+    Lower firrtl.module's to rtl.module.
+  }];
+  let constructor = "circt::firrtl::createLowerFIRRTLToRTLModulePass()";
```

Comment:
We should probably add a dependency here on the destination dialects.  I'm not totally sure what that does, or how the whole "dialect registry" thing works yet.
```
let dependencies = [rtl, sv];
```

Reply:
To answer my question, it it generates `getDependentDialects()` on the pass which is used to ensure that the dialect is loaded into the context.  The `DialectRegistry` just registers dialects so that they can be (lazily) loaded into the context when they need to be.

I'll update this to add dependentDialects.

Reply:
I added `["RTL"]` for `LowerFIRRTLToRTLModule `, and `["RTL", "SV"]` for `LowerFIRRTLToRTL`.

Reply:
Drive by comment: that sounds correct. It looks like you found this, but there is a blurb in the MLIR FAQ about this: https://mlir.llvm.org/getting_started/Faq/#registered-loaded-dependent-whats-up-with-dialects-management

Reply:
Thanks for the link!  I totally missed the doc and was just reading the code